app/src/main/java/at/roteskreuz/stopcorona/model/repositories/QuarantineRepository.kt

```diff
@@ -136,8 +141,9 @@ class QuarantineRepositoryImpl(
         private const val PREF_SHOW_QUARANTINE_END = Prefs.QUARANTINE_REPOSITORY_PREFIX + "show_quarantine_end"
     }
 
-    private var dateOfFirstMedicalConfirmation: ZonedDateTime?
+    override var dateOfFirstMedicalConfirmation: ZonedDateTime?
         by preferences.nullableZonedDateTimeSharedPreferencesProperty(PREF_DATE_OF_FIRST_MEDICAL_CONFIRMATION)
+        private set
```

Comment:
is this necessary when you set the interface-definition to `val`?

Reply:
Cool, didn't know. 👍  Updated